st2actions/tests/unit/test_runner_container.py

```diff
@@ -330,87 +327,6 @@ def test_dispatch_override_default_action_params(self):
         self.assertTrue(result.get('action_params').get('actionint') == 20)
         self.assertTrue(result.get('action_params').get('actionstr') == 'foo')
 
-    def test_state_db_created_for_polling_async_actions(self):
```

Comment:
I'm not sure why (but there may be a good reason) we've removed the test on the Polling AsyncActionRunner. That class still exists in the st2common/runners/base.py - so if the code is kept, shouldn't we keep the test that checks the logic?

Reply:
Oh sorry, that was a mistake when I started to think whether we want to drop the whole async runner. I added the tests again and all tests passed the CI.

Reply:
Resolved with 34ffaf4.